plugins/uberdev/commands/turbo.md

```
@@ -0,0 +1,334 @@
+PERM_FLAG="PERM_FLAG_VALUE"
+CLAUDE_BIN --model 'claude-opus-4-7[1m]' --effort max --worktree solve-issue-ISSUE_NUM $PERM_FLAG "$PROMPT"
+SCRIPT_EOF
+# BSD/macOS sed needs '-i ""'; GNU sed needs bare '-i'.
+SED_INPLACE=(-i '')
+[[ "$(uname)" == "Linux" ]] && SED_INPLACE=(-i)
+sed "${SED_INPLACE[@]}" "s|REPO_ROOT|$(pwd)|g" /tmp/solve-$ISSUE_NUM.sh
+REAL_CLAUDE=$(WRAPPER_DIR=$(dirname "$(which claude)"); for d in ${(s/:/)PATH}; do [[ "$d" == "$WRAPPER_DIR" ]] && continue; [[ -x "$d/claude" ]] && echo "$d/claude" && break; done)
+sed "${SED_INPLACE[@]}" "s|CLAUDE_BIN|${REAL_CLAUDE:-$(which claude)}|g" /tmp/solve-$ISSUE_NUM.sh
+sed "${SED_INPLACE[@]}" "s/ISSUE_NUM/$ISSUE_NUM/g" /tmp/solve-$ISSUE_NUM.sh
+sed "${SED_INPLACE[@]}" "s/TIER/$TIER/g" /tmp/solve-$ISSUE_NUM.sh
+# DETECTED_TERMINAL is set by Step 5.5 below; if you reorder, move that step before Step 5.
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)